gnosis/safe/api/transaction_service_api/transaction_service_api.py

```diff
@@ -168,15 +169,23 @@ def get_safe_transaction(
             safe_tx.tx_hash = tx_hash
         return safe_tx, tx_hash
 
-    def get_transactions(self, safe_address: ChecksumAddress) -> List[Dict[str, Any]]:
+    def get_transactions(
+        self, safe_address: ChecksumAddress, **kwargs: Union[str, int, bool]
```

Comment:
```suggestion
        self, safe_address: ChecksumAddress, **kwargs: Dict[str, Union[str, int, bool]]
```

Complex to annotate that 😅 : https://rednafi.com/python/annotate_args_and_kwargs/

Reply:
My mistake, I didn't pay enough attention. Good catch :)